webpack/assets/images

```diff
@@ -0,0 +1 @@
+../../app/assets/images/
```

Comment:
are we actually dealing with images here? or is it just to avoid changing the relative URLs? 
IMHO, if you do load images (css?) you need something like this in your webpack configuration as well:
```js
      { test: /\.gif(\?v=\d+\.\d+\.\d+)?$/, loader: 'file'},
      { test: /\.jpg(\?v=\d+\.\d+\.\d+)?$/, loader: 'file'},
```